packages/create-plugin/templates/common/.config/webpack/webpack.config.ts

```diff
@@ -34,7 +35,11 @@ __webpack_public_path__ =
 `,
 });
 
-const config = async (env): Promise<Configuration> => {
+export type Env = {
+  [key: string]: true | string | Env;
+};
```

Comment:
Maybe this should be?
```ts
export type Env = Record<string, boolean | string | Env>
```

Reply:
Ah yeah, they're the same (I think?), happy to change to keep it cleaner 👍

Reply:
FYI I had to revert this after checking locally, using a `Record` here causes TS to generate a circular reference error.